src/blocks/blocks/button-group/group/inspector.js

```diff
@@ -105,6 +158,28 @@ const Inspector = ({
 					] }
 					onChange={ e => setAttributes({ collapse: e }) }
 				/>
+				<ResponsiveControl
+					label={ __( 'Alignment', 'otter-blocks' ) }
+					className="buttons-alignment-control"
+				>
+					<Toolbar
+						label="Options"
+						className="buttons-alignment-toolbar"
+					>
+						{ alignmentOptions.map( option => {
+							return (
+								<ToolbarButton
+									key={ option.value }
+									icon={ option.icon }
+									title={ option.label }
+									isPrimary={ option.value === ( attributes.align[ currentDevice ] ?? 'none' ) }
+									showTooltip
+									onClick={ () => onAlignmentChange( option.value ) }
+								/>
+							);
+						}) }
+					</Toolbar>
```

Comment:
You don't need to make a new component here, there's one in Core that you can use already:

```
import { BlockAlignmentToolbar } from '@wordpress/block-editor';

<BlockAlignmentToolbar
	value={ value }
	isCollapsed={ false }
	controls={ [ 'left', 'center', 'right', 'full' ] }
	onChange={ () => {} }
/>
```